cli/dcoscli/package/main.py

```diff
@@ -308,8 +314,16 @@ def _install(package_name, options_path, app_id, cli, app, yes):
               "repositories"
         raise DCOSException(msg)
 
-    # TODO(CD): Make package version to install configurable
-    pkg_version = pkg.latest_version()
+    version_dir = [k for k, v in iteritems(pkg.software_versions())
+                   if v == pkg_version]
+    if version_dir:
+        pkg_version = ''.join(version_dir)
+    else:
+        if pkg_version and not version_dir:
+            emitter.publish(
```

Comment:
We should probably exit with return code 1 at this point.  The user as explicitly asked for something that can't be done.